ui/client/package.json

```diff
@@ -142,28 +151,32 @@
     "karma-webpack": "^4.0.0-rc.6",
     "less": "^3.9.0",
     "less-loader": "^5.0.0",
-    "lint-staged": "^9.5.0",
+    "lint-staged": "^10.0.8",
+    "nib": "^1.1.2",
     "node-libs-browser": "^0.7.0",
     "puppeteer": "^2.0.0",
     "raw-loader": "^0.5.1",
     "style-loader": "^0.23.1",
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.2",
+    "svg-transform-loader": "^2.0.8",
+    "svgo": "^1.3.2",
+    "svgo-loader": "^2.2.1",
     "terser-webpack-plugin": "^1.4.3",
-    "typescript": "^3.7.4",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.10.1",
-    "@testing-library/dom": "latest"
+    "typescript": "^3.8.2",
+    "typescript-plugin-css-modules": "^2.2.0",
+    "webpack": "^4.41.6",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.10.3"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
-    "*.js": [
-      "npm run lint -- --fix --quiet",
+    "**/*.(js|ts)?(x)": [
+      "npm run lint -- --fix",
```

Comment:
Now we will have more verbose info that lint changed our code during pre commit?